JSTutorial/hw_4/task_3.js

```diff
@@ -0,0 +1,48 @@
+	function word(wordSet) {
+		let word = wordSet.toLowerCase()
+		const vowels = "aeiou"
+		let vowels_count = 0
+		let consonants_count = 0
+	for(const i of word) {
+		if(vowels.includes(i)) {
+			vowels_count++; 
+		} else if (isNaN(i)) {
```

Comment:
Add the string that contains consonants and do `.includes(i)` check
`isNaN` would not work with special characters as `#, $` etc.